addons/polkbaskera/CfgVehicles_Units.hpp

```diff
@@ -0,0 +1,93 @@
+class B_Soldier_F;
+class CLASS(B_Rifleman_F): B_Soldier_F {
+	dlc = QUOTE(prefix);
+	scope = 2;
+	curatorScope = 2;
+	genericNames = "RussianMen";
+	identityTypes[] = {"LanguageRUS_F","Head_Russian","Head_Euro","Head_Enoch","Head_Asian","G_RUS_SF"};
+	faction = QCLASS(B_PolkBaskera);
+	weapons[] = {"arifle_AK12U_CMore", "hgun_Rook40_F", "Throw", "Put"};
+	respawnWeapons[] = {"arifle_AK12U_CMore", "hgun_Rook40_F", "Throw", "Put"};
+	magazines[] = {R6("30Rnd_762x39_Mag_Green_F"), "16Rnd_9x21_Mag","HandGrenade", R2("SmokeShell")};
+	respawnMagazines[] = {R6("30Rnd_762x39_Mag_Green_F"), "16Rnd_9x21_Mag","HandGrenade", R2("SmokeShell")};
+	linkedItems[] = {"WSLV_Platecarrier_GRN_NF", "CUP_H_RUS_6B27_headset_goggles","G_Balaclava_blk", "ItemMap", "ItemCompass", "ItemWatch", "ItemRadio"};
+	respawnLinkedItems[] = {"WSLV_Platecarrier_GRN_NF", "CUP_H_RUS_6B27_headset_goggles","G_Balaclava_blk", "ItemMap", "ItemCompass", "ItemWatch", "ItemRadio"};
+	Items[] = {R4("ACE_elasticBandage"), R4("ACE_packingBandage"), R2("ACE_quikclot")};
+	RespawnItems[] = {R4("ACE_elasticBandage"), R4("ACE_packingBandage"), R2("ACE_quikclot")};
+	uniformClass = "CUP_I_B_PMC_Unit_34";
+};
+
+class CLASS(B_TeamLeader_F): CLASS(B_Rifleman_F) {
+	displayname = "Team Leader";
+	icon = "iconManLeader";
+	weapons[] = {"arifle_AKS_F", "hgun_Rook40_F", "Binocular", "Throw", "Put"};
+	respawnWeapons[] = {"arifle_AKS_F", "hgun_Rook40_F", "Binocular", "Throw", "Put"};
+	magazines[] = {R6("30Rnd_545x39_Mag_Green_F"), R2("16Rnd_9x21_Mag"),"HandGrenade", "SmokeShell", "SmokeShellRed", "SmokeShellPurple"};
+	respawnMagazines[] = {R6("30Rnd_545x39_Mag_Green_F"), R2("16Rnd_9x21_Mag"),"HandGrenade", "SmokeShell", "SmokeShellRed", "SmokeShellPurple"};
+	linkedItems[] = {"WSLV_Platecarrier_GRN_NF", "H_Beret_blk","G_Bandanna_blk", "ItemMap", "ItemCompass", "ItemWatch", "ItemRadio", "CUP_NVG_1PN138_Hide"};
+	respawnLinkedItems[] = {"WSLV_Platecarrier_GRN_NF", "H_Beret_blk","G_Bandanna_blk", "ItemMap", "ItemCompass", "ItemWatch", "ItemRadio", "CUP_NVG_1PN138_Hide"};
+};
+
+class CLASS(B_ATRifleman_F): CLASS(B_Rifleman_F) {
+	displayname = "AT Rifleman";
+	icon = "iconManAT";
+	role = "MissileSpecialist";
+	weapons[] = {"arifle_AKM_F", "launch_RPG7_F", "Throw", "Put"};
+	respawnWeapons[] = {"arifle_AKM_F", "launch_RPG7_F", "Throw", "Put"};
+	magazines[] = {R5("30Rnd_762x39_Mag_Green_F"), R3("RPG7_F"), R2("SmokeShell")};
+	respawnMagazines[] = {R5("30Rnd_762x39_Mag_Green_F"), R3("RPG7_F"), R2("SmokeShell")};
```

Comment:
Only need to add one, for the weapon
```suggestion
	magazines[] = {R5("30Rnd_762x39_Mag_Green_F"), "RPG7_F", R2("SmokeShell")};
	respawnMagazines[] = {R5("30Rnd_762x39_Mag_Green_F"), "RPG7_F", R2("SmokeShell")};
```